packages/@o3r/eslint-plugin/package.json

```
@@ -45,12 +45,12 @@
     }
   },
   "peerDependencies": {
-    "@angular-devkit/schematics": "~18.2.0",
-    "@angular-eslint/template-parser": "~18.4.0",
-    "@angular-eslint/test-utils": "~18.4.0",
-    "@angular/compiler": "~18.2.0",
-    "eslint": "^8.57.0 || ~9.14.0",
-    "jsonc-eslint-parser": "~2.4.0",
+    "@angular-devkit/schematics": "^18.2.0",
+    "@angular-eslint/template-parser": "^18.4.0",
+    "@angular-eslint/test-utils": "^18.4.0",
+    "@angular/compiler": "^18.2.0",
+    "eslint": "^8.57.0 || ^9.14.0",
```

```suggestion
    "eslint": ">=8.57.0 <9.15.0",
```
to be aligned with @o3r/eslint-config, it's needed because there is an issue with 9.15 at least